site/content/docs/5.2/examples/stickers/index.html

```diff
@@ -7,7 +7,7 @@
 ---
 
 <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
-  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="5G">
+  <symbol fill="currentColor" viewBox="0 0 1000 1000" id="fiveG">
```

Comment:
Should we change the `boosted-sprite.svg` that uses the same id (beginning with a number)?

Reply:
Apparently that's not an issue in a SVG file so let's not change anything